crates/bevy_ecs/src/archetype.rs

```diff
@@ -153,16 +176,18 @@ impl Edges {
 }
 
 pub struct ArchetypeEntity {
```

Comment:
Why does this need to be pub?

Reply:
It's currently used by the scenes API. I should double check if we can just change that to using an iterator instead of a raw slice, but the data there cannot be hidden away trivially without #6681.

Reply:
Added temp placeholder docs until this either gets hidden or we decide otherwise.